src/llm/llm_engine.py

```
@@ -0,0 +1,379 @@
+            top_p: Top-p sampling parameter (uses config default if None)
+            top_k: Top-k sampling parameter (uses config default if None)
+            stop_sequences: List of sequences that stop generation
+            
+        Returns:
+            str: Generated response text
+            
+        Raises:
+            RuntimeError: If model is not loaded
+        """
+        if not self.is_model_loaded():
+            raise RuntimeError("Model is not loaded. Call load_model() first.")
```

Ruff (0.13.1)

Avoid specifying long messages outside the exception class

(TRY003)